apikit/convenience.py

```diff
@@ -335,6 +423,25 @@ def __init__(self, name, version, repository, description,
                            api_version=api_version,
                            auth=auth,
                            route=route)
+        logfile = None
+        syslog = False
+        loghost = None
+        loglevel = None
+        if "LOGFILE" in os.environ and os.environ["LOGFILE"]:
+            logfile = os.environ["LOGFILE"]
+        elif "LOG_TO_SYSLOG" in os.environ and os.environ["LOG_TO_SYSLOG"]:
+            syslog = True
+            if "LOGHOST" in os.environ and os.environ["LOGHOST"]:
+                loghost = os.environ["LOGHOST"]
+        if "LOGLEVEL" in os.environ and os.environ["LOGLEVEL"]:
+            loglevel = os.environ["LOGLEVEL"]
+        if "DEBUG" in os.environ and os.environ["DEBUG"]:
+            self.debug = True
+            self.config["DEBUG"] = True
+            loglevel = "DEBUG"
+        log = get_logger(file=logfile, syslog=syslog, loghost=loghost,
+                         level=loglevel)
+        self.config["LOGGER"] = log
```

Comment:
Can't this be done through just importing a logging class combined with a singleton or some similar class shenanigans?